src/UrlRpcEncoder.ts

```diff
@@ -144,7 +155,7 @@ export class UrlRpcEncoder {
         } else {
             url.hash = fragment.toString();
         }
-
-        history.replaceState(history.state, '', url.href);
     }
+
 }
+
```

Comment:
Terrific lines. The best lines. There's no lines better than these!

**MAKE LINES GREAT AGAIN!** :trumpet: